tldr.py

```diff
@@ -224,25 +224,7 @@ def output(page):
         print()
 
 
-def update_cache(remote=None):
-    cache_path = get_cache_dir()
-    if not os.path.exists(cache_path):
-        return
-    files = [file_name for file_name in os.listdir(cache_path)
-             if os.path.isfile(os.path.join(cache_path, file_name)) and
-             COMMAND_FILE_REGEX.match(file_name)]
-    for file_name in files:
-        match = COMMAND_FILE_REGEX.match(file_name)
-        command = match.group('command')
-        platform = match.group('platform')
-        try:
-            download_and_store_page_for_platform(command, platform, remote=remote)
-            print('Updated cache for %s (%s) from %s' % (command, platform, remote))
-        except Exception:
-            sys.exit('Error: Unable to get %s (%s) from %s' % (command, platform, remote))
```

Comment:
you can delete the `COMMAND_FILE_REGEX` variable as well since this was the only place it was used.